### PR TITLE
Signs: do not exceed two cells

### PR DESCRIPTION
**Pull request by mhinz**

### CODE REVIEW

**Comment by halostatue** [comment]
This change works and prevents the E239. As a future feature enhancement, you may want to support multiple signs for these values, possibly using `type()` to allow for `let g:signify_sign_changedelete = [ '💥', '⨂' ]` so that the second will be used for single-width changes.

**Comment by mhinz** [comment]
I'm rather reluctant to add something like this, because it's a tad too magical.

Yes, it's certainly possible, but even writing the documentation would be hard without confusing users more than actually needed.

In this case, I'll go with:

_Vim has limitations regarding signs, so choose wisely. A simple character gets the job done as well. :]_

File: doc/signify.txt [hunk 1]
```diff
@@ -325,16 +325,11 @@ Enable line highlighting in addition to using signs by default.
 The sign to use if a line was added, deleted or changed or a combination of
 these.
 
-NOTE: You can use unicode characters.
+You can use unicode characters, but signs mustn't take up more than two cells.
```
Comment (jamessan) [inline doc/signify.txt]: I'd change `mustn't` to `must not`, since I generally try to avoid contractions for documentation.  Similarly for `you'll`.

**Comment by mhinz** [comment]
@halostatue Thanks for bringing this up!

**Pull request merged**